### ISSUE TITLE
The actual connection timeout exceeds specified value twice

### ISSUE THREAD

**Issue by lospejos** [jwagenleitner/groovy-wslite#138]
The actual connection timeout exceeds specified value twice.
Groovy test code is here: [TestConnTimeout.zip](https://github.com/jwagenleitner/groovy-wslite/files/779731/TestConnTimeout.zip)

Results:
$ groovy TestConnTimeout.groovy
Started request to http://www.google.com:81
Error: wslite.soap.SOAPClientException: connect timed out
Declared timeout: 2000 ms, fact timeout: 4413 ms

**Comment by jaysamson03** [comment]
Hi! Im having the same issue. It seems that the read timeout setting is not applied. Did you have a resolution to this issue?

Thanks

**Comment by gastonmanoli** [comment]
same issue, any update?

**Comment by lospejos** [comment]
> Hi! Im having the same issue. It seems that the read timeout setting is not applied. Did you have a resolution to this issue?
> 
> Thanks



> same issue, any update?

No. I've changed my job and this issue is not critical for me now. 
But I do not close this issue because other people could have this issue.
